### ISSUE TITLE
Suggestion: Space Stations

### ISSUE THREAD

**Issue by user3647** [lightning2401/northstar#102]
**Space Stations**
Firstly, you would have to put a different block than "rocket station" in your rocket, some sort of space station controller, then you would orbit around whatever planet you go to instead of landing. 
Being a space station, it would not be able to land, so you could be able to craft some sort of mini lander shuttle that you could place down inside your station and get inside of it to land on the planet(only if you can land on it of course). 
This shuttle could be something like Ad Astra's rockets, where it is an entity that you place on a launch pad.

**Spacewalks**
You could also add a jetpack(one for each type of space suit). 
This would count as a space suit chest plate, but would also need to be filled with fuel.
This fuel could be any rocket fuel, liquid hydrocarbons, liquid methane, hydrogen, and liquid hydrogen.
You would need to fill this in a "fuel filler" first then fill the oxygen in an "oxygen filler."
This could work either like creative flight, elytra, or switchable between these modes.

**Comment by LostGameDev** [comment]
alternatively once you orbit a planet the space station "lands" (becomes normal blocks while in orbit), and you just build normal rockets instead as i think it would be more fitting considering the rest of the mod, also would be cool to make custom shuttles instead of just the normal Ad Astra style rockets that most other space mods use

**Comment by user3647** [comment]
Yes I meant it would become normal blocks when it’s in orbit. Maybe you should have a choice to make custom shuttle or use pre made like ad astra
